english/talk_to_neighbour/woman/go-with-scully/save-the-world/ride-with-scully-and-alien.md

```diff
@@ -0,0 +1,3 @@
+Ok, says Scully, for your safety, you will have to ride with us to the secret facility. Now get in the trunk!
+What?! - you say.
+It is for your own safety - replies Scully - besides, there is a micro-bar in there, help yourself. We have a long journey ahead!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.